bin/oneclick_library_delta

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+"""Update the circulation manager server with new books from the
+open access content server."""
+import os
+import sys
+bin_dir = os.path.split(__file__)[0]
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+from core.scripts import OneClickDeltaScript
+
+from api.config import Configuration
+Configuration.load()
+library_id = Configuration.integration(Configuration.ONECLICK_INTEGRATION).get("library_id", None)
+
+print "Applying catalog delta of library #%s" % library_id
+OneClickDeltaScript().run()
```

Comment:
I've seen this print statement in another script, and I liked it.  It lets the caller know that something is happening, before we try to get into the actual Script class, and right on the command line.  I agree that most stuff should be put neatly away in logs, but for this one statement, I kinda like it nice and visible to stdout?